op-node/node/node.go

```diff
@@ -575,6 +613,18 @@ func (n *OpNode) PublishL2Payload(ctx context.Context, envelope *eth.ExecutionPa
 	return nil
 }
 
+func (n *OpNode) PublishL2Attributes(ctx context.Context, attrs *derive.AttributesWithParent) error {
+	builderAttrs := attrs.ToBuilderPayloadAttributes()
+	jsonBytes, err := json.Marshal(builderAttrs)
+	if err != nil {
+		n.log.Warn("failed to marshal payload attributes", "err", err)
+		return err
+	}
+	n.log.Info("Publishing execution payload attributes on event stream", "attrs", builderAttrs, "json", string(jsonBytes))
```

Comment:
is `"json", string(jsonBytes)` left from debugging? logs would be quite verbose

Reply:
45c01647f2a59b662f627f9398fc40c42d4b9e0f

Reply:
45c01647f2a59b662f627f9398fc40c42d4b9e0f

Reply:
9b66f1d6c0c6a046b3b84c0eb86ce3597fd6baaa